compiler/rustc_session/src/session.rs

```diff
@@ -1362,23 +1362,16 @@ pub struct EarlyDiagCtxt {
     dcx: DiagCtxt,
 }
 
-impl Default for EarlyDiagCtxt {
-    fn default() -> Self {
-        Self::new(ErrorOutputType::default())
```

Comment:
this reverted https://github.com/rust-lang/rust/pull/133757/ :/ was that intentional?

Reply:
Probably not. It had no comment and no internal users, thus it got stripped